front-end/src/Home.js

```diff
@@ -1,15 +1,13 @@
 import React from 'react'
+import Header from './Header'
 import Footer from './Footer'
-import Hamburger from './Hamburger'
 import './Home.css'
 
 function Home() {
     return (
 	<>
-	    <Hamburger pageWrapId={'page-wrap'} outerContainerId={'outer-container'} />
-	    <div className = "header">
-		<h1>Welcome to TV Tracker!</h1>
-	    </div>
+		<Header />
+
 	    <img src="https://picsum.photos/1150/300" alt="Temporary from lorem picsum" class="fullIMG" />
```

Comment:
This isn't anything you did, but when you make changes could you change the `class` attribute on the `img` tags in `Home.js` to be `className` instead? I missed it in my initial review.

Also, if you don't mind changing the class name itself to be something like `full-img` (here and in `Home.css`), I think it'd make the code look a bit more consistent.

Reply:
idt I touched the img tags or anything like that. we ryou referring to that?

Reply:
Sam, for what it's worth, I think using camelCase instead of hyphens looks better. We should decide, as a team, our conventions for styling (I know that text in there doesn't use camelCase exactly; fullImg would be better)